clients/admin-ui/src/types/api/models/ConnectionConfigurationResponse.ts

```diff
@@ -11,18 +11,18 @@ import type { SaaSConfigBase } from "./SaaSConfigBase";
  * Describes the returned schema for a ConnectionConfiguration.
  */
 export type ConnectionConfigurationResponse = {
-  name?: string;
+  name?: string | null;
   key: string;
-  description?: string;
+  description?: string | null;
   connection_type: ConnectionType;
   access: AccessLevel;
   created_at: string;
-  updated_at?: string;
-  disabled?: boolean;
-  last_test_timestamp?: string;
-  last_test_succeeded?: boolean;
-  saas_config?: SaaSConfigBase;
-  secrets?: any;
-  authorized?: boolean;
-  enabled_actions?: Array<ActionType>;
+  updated_at?: string | null;
+  disabled?: boolean | null;
+  last_test_timestamp?: string | null;
+  last_test_succeeded?: boolean | null;
+  saas_config?: SaaSConfigBase | null;
+  secrets?: null;
```

Comment:
here's another instance of pydantic using `null`. I'm changing this back to `any` for now

Reply:
keep as `null` if tests pass

Reply:
got it 👍